compositor_render/src/renderer/renderers.rs

```diff
@@ -26,6 +28,8 @@ impl Renderers {
             web_renderers: RendererRegistry::new(RegistryType::WebRenderer),
             images: RendererRegistry::new(RegistryType::Image),
             builtin: BuiltinTransformations::new(&wgpu_ctx)?,
+            layout: LayoutRenderer::new(&wgpu_ctx)
+                .map_err(InitBuiltinError::ApplyTransformationMatrix)?,
```

Comment:
It uses `apply_layouts.wgsl`. This error suggests that compiling `apply_transformation_matrix.wgsl` failed.